src/data/markdown/docs/40 xk6-disruptor/01 Get started/04 Exposing  Your Application.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Exposing your application'
```

Comment:
Our style guide recommends avoiding gerunds in headings: https://developers.google.com/style/headings

Reply:
In this case I wanted to communicate that this is an optional step. "Expose your application" sounds to me like a necessary step. Maybe "How to Expose your application"?

Reply:
For this, I'd prefer that we keep to guidelines, because we also have to think about standardizing the overall information environment of not only k6, but all Grafana docs.

Reading that page again, I think we it might be more specific to title it "Assign external IP". And if it's optional, this could be mentioned at the beginning. WDYT?